src/wallet/salvage.cpp

```diff
@@ -15,13 +16,13 @@ static const char *HEADER_END = "HEADER=END";
 static const char *DATA_END = "DATA=END";
 typedef std::pair<std::vector<unsigned char>, std::vector<unsigned char> > KeyValPair;
 
-bool RecoverDatabaseFile(const fs::path& file_path)
+bool RecoverDatabaseFile(const fs::path& file_path, bilingual_str& error, std::vector<bilingual_str>& warnings)
 {
     std::string filename;
     std::shared_ptr<BerkeleyEnvironment> env = GetWalletEnv(file_path, filename);
 
     if (!env->Open(true /* retry */)) {
-        tfm::format(std::cerr, "Error initializing wallet database environment %s!", env->Directory());
+        error = strprintf(_("Error initializing wallet database environment %s!"), env->Directory());
```

Comment:
Any reason to ask translators to waste effort when these were previously untranslated and the new translations are unused?

Reply:
Since these are utility functions, we might add them to the gui in the future. Or maybe add a gui to the wallettool to make it more user friendly.

Reply:
I'd prefer to err on the side of not adding a translation message if it's not currently used. This can always be done when necessary.

Reply:
I've changed these to use `Untranslated`.